pkg/controllers/cloud/networkpolicy_controller.go

```diff
@@ -62,10 +61,16 @@ const (
 	npSyncReadyBookMarkCnt = 3
 )
 
+var lastSyncTime int64
+
 // +kubebuilder:rbac:groups=controlplane.antrea.io,resources=networkpolicies,verbs=get;list;watch
 // +kubebuilder:rbac:groups=controlplane.antrea.io,resources=addressgroups,verbs=get;list;watch
 // +kubebuilder:rbac:groups=controlplane.antrea.io,resources=appliedtogroups,verbs=get;list;watch
 
+func SetLastSyncTime() {
```

Comment:
nit:accessed within the package. Change it to setLastSyncTime.

Reply:
this is wrong.. why you setting LastSyncTime here.. Config Controller shouldnt have any business logic of how fields will be used.. you need to call NetworkPolicyReconciler and set newCloudSyncInterval and reset previousTime

Reply:
Solved.